pom.xml

```diff
@@ -38,6 +38,8 @@
         <module>bricks-assemble</module>
         <module>bricks-dashboard</module>
         <module>bricks-maven-plugin</module>
+        <module>example/bricks-example-component</module>
+        <module>example/bricks-example-custom-analyzers</module>
```

Comment:
* Would like to rename the example folder to bricks-examples. 
* Makes it sense to put another pom in the bricks-examples folder?
* Are both subfolders not both containing stuff for one example? I mean, are these to different examples or do we have one example atm?